core/services/ardupilot_manager/frontend/index.html

```diff
@@ -0,0 +1,236 @@
+                        <div class="d-flex flex-justify-start">{{ endpoint.argument }}</div>
+                    </div>
+                    <div class="cell-1">
+                        <input type="checkbox" data-role="checkbox" :checked="endpoint.persistent" class="d-flex flex-justify-center" disabled />
+                    </div>
+                    <div class="cell-1">
+                        <input type="checkbox" data-role="checkbox" :checked="endpoint.protected" class="d-flex flex-justify-center" disabled />
+                    </div>
+                    <div class="cell-1">
+                        <button class="button primary alert cycle" @click="deleteEndpoint(endpoint)">
+                          <span class="mif-minus"></span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+
+            <div id="infobox" class="info-box" data-role="infobox">
+                <span class="button square closer"></span>
+                <div class="info-box-content">
+                    <h3>{{ infoboxTitle }}</h3>
+                    <p>{{ infoboxMessage }}</p>
+                </div>
+            </div>
+
+        </div>
+    </div>
+</body>
+
+<script>
+    /*global Metro axios, Vue*/
+
+    new Vue({
+      el: '#app',
+      data: function () {
+        return {
+          newEndpoint: {
+            name: '',
+            owner: '',
+            connectionType: '',
+            place: '',
+            argument: null,
+            persistent: false,
+            protected: false,
+          },
+          registeredEndpoints: [],
+          Metro: null,
+          infoboxTitle: '',
+          infoboxMessage: '',
+        }
+      },
+      created () {
+        this.API_URL = '/latest/endpoints'
+        this.connectionTypeOptions = {
+          serial: 'Serial',
+          udpin: 'UDP Server',
+          udpout: 'UDP Client',
+          tcpin: 'TCP Server',
+          tcpout: 'TCP Client',
+        }
+      },
+      mounted () {
+        Metro.init()
+        this.Metro = Metro
+        this.fetchRegisteredEndpoints()
+        setInterval(this.fetchRegisteredEndpoints, 5000)
+      },
+      methods: {
+        parseEndpointFromAPI (endpoint) {
+          return {
+            name: endpoint.name,
+            owner: endpoint.owner,
+            connectionType: endpoint.connection_type,
+            place: endpoint.place,
+            argument: parseInt(endpoint.argument),
+            persistent: endpoint.persistent,
+            protected: endpoint.protected,
+          }
+        },
+        parseEndpointToAPI (endpoint) {
+          return {
+            'name': endpoint.name,
+            'owner': endpoint.owner,
+            'connection_type': endpoint.connectionType,
+            'place': endpoint.place,
+            'argument': endpoint.argument,
+            'persistent': endpoint.persistent,
+            'protected': endpoint.protected,
+          }
```

Comment:
All of that is necessary because of the camelCase ?

Reply:
Yes

Reply:
It's better to avoid such thing, if linter is a problem we should fix it. The amount of code for such thing is terrible for software development, should be fixed asap.

Reply:
Agree. Will wait for #223 to be merged so I send the update.